int32 action_id  # 0: Move forward, 1: Turn left, 2: Turn right, 3: Stop
